index.js

```diff
@@ -184,6 +207,12 @@ var Umzug = module.exports = redefine.Class({
     }
   },
 
+  log: function() {
+    if (this.options.logging) {
+      this.options.logging.apply(null, [].slice.call(arguments));
```

Comment:
I'm not sure I fully understand your question. The consumer could potentially add their very own logging function that takes multiple arguments, or even want to pass multiple arguments into console.log. In both cases I would expect that .log passes them on 1-1 to the passed logging function. Does that answer your question?